tests/data/packages/BrokenEmitsUTF8/setup.py

```diff
@@ -10,14 +8,10 @@ class FakeError(Exception):
 
 if sys.argv[1] in ("install", "bdist_wheel"):
     if hasattr(sys.stdout, "buffer"):
+        sys.stdout.buffer.write(b"\nThis package prints out UTF-8 stuff like:\n")
```

Comment:
```suggestion
        sys.stdout.buffer.write("\nThis package prints out UTF-8 stuff like:\n".encode())
```

I prefer this for consistency, so if someone wants to click "apply" that's fine. But I don't care that much, so ignore it if you prefer.